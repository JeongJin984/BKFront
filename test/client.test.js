import withReduxSaga from '..'

import FunctionalComponent from './components/functional-component'

import wrapper from './store/store-wrapper'

import createSnapshot from './utils/create-snapshot'

test('Wrapped component passes along React props', () => {
    const WrappedComponent = wrapper.withRedux(
        withReduxSaga(FunctionalComponent),
    )

    createSnapshot(WrappedComponent)
})