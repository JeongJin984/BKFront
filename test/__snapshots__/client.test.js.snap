// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapped component passes along React props 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "@@observable": [Function],
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "sagaTask": Object {
          "@@redux-saga/TASK": true,
          "cancel": [Function],
          "cont": [Function],
          "context": Object {},
          "end": [Function],
          "error": [Function],
          "id": 1,
          "isAborted": [Function],
          "isCancelled": [Function],
          "isRoot": true,
          "isRunning": [Function],
          "joiners": Array [],
          "meta": Object {
            "location": undefined,
            "name": "rootSaga",
          },
          "queue": Object {
            "abort": [Function],
            "addTask": [Function],
            "cancelAll": [Function],
            "getTasks": [Function],
          },
          "result": [Function],
          "setContext": [Function],
          "toPromise": [Function],
        },
        "subscribe": [Function],
      },
      "subscription": Subscription {
        "handleChangeWrapper": [Function],
        "listeners": Object {
          "notify": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "@@observable": [Function],
          "dispatch": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "sagaTask": Object {
            "@@redux-saga/TASK": true,
            "cancel": [Function],
            "cont": [Function],
            "context": Object {},
            "end": [Function],
            "error": [Function],
            "id": 1,
            "isAborted": [Function],
            "isCancelled": [Function],
            "isRoot": true,
            "isRunning": [Function],
            "joiners": Array [],
            "meta": Object {
              "location": undefined,
              "name": "rootSaga",
            },
            "queue": Object {
              "abort": [Function],
              "addTask": [Function],
              "cancelAll": [Function],
              "getTasks": [Function],
            },
            "result": [Function],
            "setContext": [Function],
            "toPromise": [Function],
          },
          "subscribe": [Function],
        },
        "unsubscribe": null,
      },
    }
  }
>
  <withReduxSaga(Connect(FunctionalComponent)) />
</ContextProvider>
`;
